python/cherrypy/requirements.txt

```diff
@@ -0,0 +1,2 @@
+cherrypy>=18.3.0
```

Comment:
you must set also `<19.4.0` in order to ensure to be on **18.3**
~~~python
cherrypy>=18.3.0,<19.4.0
~~~